### DISCUSSION TITLE
Assigning issues, not just PRs, to teams

### DISCUSSION THREAD

**Discussion by gwynne** [community/community#57306] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

It has been possible for some time now to assign pull requests to entire teams, such as via `CODEOWNERS`. However, it is not possible to do the same with issues. As seen in https://github.com/dear-github/dear-github/issues/170, many have been looking for this feature for a long time, with no apparent feedback I've been able to find. Is there a reason this hasn't been implemented, whether technical or otherwise?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by zsloane** [comment]
+1

This would be a really useful addition

**Comment by mbtre** [comment]
Was looking for this everywhere. Cannot believe that this is not possible. Doesn't this defy the whole purpose of teams?
Please implement this!

**Comment by Danielku15** [comment]
Just ran into the same finding that this is not possible and would like to see this. I guess I was confused by the fact that teams can be reviewers on PRs but they also cannot be assignees there. But from a workflow it makes sense that sometimes anyone from a team can take care of a PR or Issue for the project. This makes it easier to distribute the workload. Especially as I would like to define the team as assignee via issue templates.  

I found following discussions as possible duplicates: 

- https://github.com/orgs/community/discussions/86679
- https://github.com/orgs/community/discussions/47450
- https://github.com/orgs/community/discussions/5658
- https://github.com/orgs/community/discussions/12087
